.github/workflows/build.yml

```diff
@@ -15,38 +14,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-      - name: Checkout code
-        id: checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-          submodules: true
+    - name: Checkout code
+      id: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: true
```

Comment:
Do we really?

Reply:
I don't think so, but is there a downside to including this (especially as it was already in)?

Reply:
The downside to including this is that builds will succeed if we incorrectly add a git submodule to the repo 🤡